src/com/dhbw/MorseProject/MorseProject.java

```diff
@@ -1,4 +1,64 @@
 package com.dhbw.MorseProject;
 
+import com.dhbw.MorseProject.send.Encoder;
+import com.dhbw.MorseProject.send.Melody;
+import com.dhbw.MorseProject.translate.Translator;
+
+// todo comment ids from Pflichtenheft ;) where these are implemented
+// todo setup jar export
```

Comment:
TODOs in main branch are not common, but it should be okay as this has nothing to do with the code itself.

Reply:
better than no code ;)